### ISSUE TITLE
New content: Page-level keyboard shortcuts

### ISSUE THREAD

**Issue by Ozyugoo** [wagtail/guide#346]
I think that keyboard shortcuts on page level needs to be documented centering on how to navigate the page with keyboard shortcuts for all users.

**Comment by Ozyugoo** [comment]
This document provides comprehensive information about the keyboard shortcuts available on our website, enhancing your user experience and productivity. These shortcuts allow you to navigate the website efficiently without relying solely on your mouse or touchpad.

## Keyboard Navigation
Users can navigate the web pages and interact with elements using the keyboard. One can access the functions without the use of a mouse.

## Some basic keyboard shortcuts
The **Tab Key** moves forward through the elements on the page such as links, buttons, and form fields. Use the Shift + Tab to move backward through the elements.
Use the **Spacebar** to activate buttons or links that are on focus.
Use the **Arrow keys** (up and down) to scroll through the content.

### Website Navigation
**Access Links**: The **Tab key** navigates and keeps the links in focus. When the link is on focus, Press **Enter** to follow the link.
**Form Fields**: Use the **Tab key** to move through form fields, keep the input field on focus, and select options in dropdown menus, etc.
Use **Enter** will submit the form.
**Menus and Dropdowns**: Press **Enter** to open the menu. Use the **Tab keys** to go through the menu items, and press **Enter** to select an item.
Users can access Wagtail websites using browser Shortcuts:
Ctrl + L (Cmd + L on Mac): Focus the browser's address bar, allowing you to type a new URL.
Ctrl + W (Cmd + W on Mac): Close the current tab.
Ctrl + T (Cmd + T on Mac): Open a new tab.
Ctrl + Tab (Cmd + Option + Right Arrow on Mac): Switch between open tabs.
Ctrl + Shift + T (Cmd + Shift + T on Mac): Reopen the last closed tab.

**Comment by Ozyugoo** [comment]
>@thibaudcolas Please kindly review.

**Comment by lb-** [comment]
I personally don't think we need to document generic browser shortcuts like `tab`. 

However, I think we could document some shortcuts for the rich text editor.

However, it's probably best for these to be provided canonically in the UI itself. Not in the guide. 

See the following issues for the planning/requests for these. 

- https://github.com/wagtail/wagtail/issues/3949 - main one
- https://github.com/wagtail/wagtail/issues/10167
- https://github.com/wagtail/wagtail/issues/9366 ( a bug)

**Comment by Ozyugoo** [comment]
> I personally don't think we need to document generic browser shortcuts like `tab`.
> 
> However, I think we could document some shortcuts for the rich text editor.
> 
> However, it's probably best for these to be provided canonically in the UI itself. Not in the guide.
> 
> See the following issues for the planning/requests for these.
> 
> * [Keyboard shortcut documentation for editor in the Wagtail UI wagtail#3949](https://github.com/wagtail/wagtail/issues/3949) - main one
> * [🎛️ Replace MouseTrap usage with built in Stimulus keyboard support wagtail#10167](https://github.com/wagtail/wagtail/issues/10167)
> * [Draftail (rich text editor) - using cmd + left arrow to navigate to the start of the line is broken in Firefox wagtail#9366](https://github.com/wagtail/wagtail/issues/9366) ( a bug)

> Thank you, I will read through this links and come with a better idea.

**Comment by thibaudcolas** [comment]
@Ozyugoo as LB mentions it’s not terribly useful to us to document very generic keyboard commands. This is documentation for Wagtail, not for how internet browsers or operating systems work.

I’d agree it’d be best for those shortcuts to be documented in the UI but this isn’t mutually exclusive, so let’s have docs in the guide until we’re in a position to have them elsewhere instead or in addition.

**Comment by lb-** [comment]
I agree with Thibaud, nothing's stopping us from having the keyboard shortcuts listed in the Guide and also within the UI itself. 

Just need to make sure they are kept in sync if they change / are added to.

**Comment by lb-** [comment]
We may also want to document the usage of `shift+click` for multiple areas in the UI, usually for large listings of check boxes.

* Images (bulk actions)
* Documents (bulk actions)
* Form submission listings (bulk delete)
* Snippet listings (bulk actions)
* Locale listings (for page copy, a bit obscure though)
* Maybe: Group permissions - https://github.com/wagtail/wagtail/pull/10867

**Comment by Ozyugoo** [comment]
> We may also want to document the usage of `shift+click` for multiple areas in the UI, usually for large listings of check boxes.
> 
> * Images (bulk actions)
> * Documents (bulk actions)
> * Form submission listings (bulk delete)
> * Snippet listings (bulk actions)
> * Locale listings (for page copy, a bit obscure though)
> * Maybe: Group permissions - [Add bulk toggling of Object Permissions in the User Group edit page wagtail#10867](https://github.com/wagtail/wagtail/pull/10867)

> Thank you for tips, I will take time to study it and document something.

**Comment by Ozyugoo** [comment]
> @Ozyugoo as LB mentions it’s not terribly useful to us to document very generic keyboard commands. This is documentation for Wagtail, not for how internet browsers or operating systems work.
> 
> I’d agree it’d be best for those shortcuts to be documented in the UI but this isn’t mutually exclusive, so let’s have docs in the guide until we’re in a position to have them elsewhere instead or in addition.

> Noted, thank you for the feedback

**Comment by thibaudcolas** [comment]
Now done on https://guide.wagtail.org/en-latest/concepts/accessibility-features/#page-level-keyboard-shortcuts
